workflows4s-core/src/test/scala/workflows4s/runtime/InMemorySyncRuntimeTest.scala

```diff
@@ -1,20 +1,21 @@
 package workflows4s.runtime
 
+import cats.Id
 import org.scalatest.freespec.AnyFreeSpec
 import workflows4s.runtime.instanceengine.WorkflowInstanceEngine
 
 class InMemorySyncRuntimeTest extends AnyFreeSpec {
 
-  import workflows4s.wio.TestCtx.*
+  import workflows4s.wio.TestCtx.{*, given}
 
-  "InMemorySyncRuntime" - {
+  "InMemoryRuntime with Id effect" - {
```

Comment:
I think this test now needs to be renamed/merged. There is nothing specific about sync behavior here, the whole inMemory runitme should follow this behavior.